### PR TITLE
Update to VS2015/.NET 4.6

### PR DESCRIPTION
**Pull request by JasonLawrenceDuncan**
Update to VS2015/.NET 4.6

Will update version # /description directly on master.

### CODE REVIEW

File: Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (stevehaselschwerdt) [inline Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj]: nit: update to 14

File: Vimeo-DotNet/VimeoAPI/VimeoDotNet.nuspec [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <iconUrl>http://assets.techsmith.com/Images/content/mkt-about/tsc-dl-image.png</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>$description$</description>
-    <releaseNotes>Initial NuGet package</releaseNotes>
+    <releaseNotes></releaseNotes>
```
Comment (stevehaselschwerdt) [inline Vimeo-DotNet/VimeoAPI/VimeoDotNet.nuspec]: You can just delete this altogether.

File: Vimeo-DotNet/VimeoAPI/packages.config [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="5.0.5" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
```
Comment (stevehaselschwerdt) [inline Vimeo-DotNet/VimeoAPI/packages.config]: We're trying to update `Newtonsoft.Json` to 7.0.1(It's in the Trello card template for reference). If you can't find it for some reason, make sure you're pointing at the `nuget` source and not the `TechSmith` source.

File: Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj (hunk 1 above)
Comment (JasonLawrenceDuncan) [inline Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj]: Fixed in 8e30e0e.

File: Vimeo-DotNet/VimeoAPI/VimeoDotNet.nuspec (hunk 2 above)
Comment (JasonLawrenceDuncan) [inline Vimeo-DotNet/VimeoAPI/VimeoDotNet.nuspec]: Deleted in 1410b6e.

File: Vimeo-DotNet/VimeoAPI/packages.config (hunk 3 above)
Comment (JasonLawrenceDuncan) [inline Vimeo-DotNet/VimeoAPI/packages.config]: Yep - I had pulled the most recent version from TechSmith, not NuGet: fixed in 40f1a31.

Comment (JasonLawrenceDuncan) [inline Vimeo-DotNet/VimeoAPI/packages.config]: Something had gone awry, so I removed it and reinstalled it, to get the update in the PROJ file, too. Fixed in 081eb73.

**Comment by stevehaselschwerdt** [comment]
:+1:

**Pull request merged**